evm-template/runtime/src/weights/pallet_balances.rs

```diff
@@ -68,6 +68,28 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+	//	TODO: below is random
```

Comment:
@KitHat I had to add these random entries to make the compiler happy. Do you think we should address these random values immediately, or they can wait till the next time we update our benchmarks?

Reply:
We can leave them for now and address them during the release process